documentation/modules/ROOT/pages/configuration/storage.adoc

```diff
@@ -0,0 +1,635 @@
+|No default
+|Must be set to `org.apache.kafka.connect.storage.KafkaOffsetBackingStore`.
+
+|[[offset-storage-topic]]<<offset-storage-topic,`offset.storage.topic`>>
+|No default
+|Defines the Kafka topic where offsets will be stored. It's essential that this topic is configured with log compaction enabled to retain the latest offset information.
+
+|[[offset-storage-partitions]]<<offset-storage-partitions,`offset.storage.partitions`>>
+|25
+|Indicates the number of partitions for the offset storage topic. This should align with your Kafka cluster's partitioning strategy.
+
+|[[offset-storage-replication-factor]]<<offset-storage-replication-factor,`offset.storage.replication.factor`>>
+|3
+|Sets the replication factor for the offset storage topic, ensuring fault tolerance by replicating data across multiple brokers.
+
+|===
+
+=== Internal schema history store
+
+[cols="35%a,10%a,55%a",options="header"]
+|===
+|Property
+|Default
+|Description
+
+|[[schema-history-internal-kafka]]<<schema-history-internal-kafka,`schema.history.internal.kafka`>>
+|No default
+|Must be set to `io.debezium.storage.kafka.history.KafkaSchemaHistory`.
+
+|[[schema-history-internal-kafka-topic]]<<schema-history-internal-kafka-topic,`schema.history.internal.kafka.topic`>>
+|No default
+|The name of the topic for the database schema history
+
+|[[schema-history-internal-kafka-bootstrap-servers]]<<schema-history-internal-kafka-bootstrap-servers,`schema.history.internal.kafka.bootstrap.servers`>>
+|No default
+|A list of host/port pairs that the connector will use for establishing the initial connection to the Kafka cluster for retrieving database schema history previously stored by the connector. This should point to the same Kafka cluster used by the Kafka Connect process.
+
+|[[schema-history-internal-kafka-recovery-poll-interval-ms]]<<schema-history-internal-kafka-recovery-poll-interval-ms,`schema.history.internal.kafka.recovery.poll.interval.ms`>>
+|100
+|The number of milliseconds to wait while polling for persisted data during recovery.
+
+|[[schema-history-internal-kafka-recovery-attempts]]<<schema-history-internal-kafka-recovery-attempts,`schema.history.internal.kafka.recovery.attempts`>>
+|100
+|The number of attempts in a row that no data are returned from Kafka before recover completes. The maximum amount of time to wait after receiving no data is (recovery.attempts) x (recovery.poll.interval.ms).
+
+|[[schema-history-internal-kafka-query-timeout-ms]]<<schema-history-internal-kafka-query-timeout-ms,`schema.history.internal.kafka.query.timeout.ms`>>
+|3
+|The number of milliseconds to wait while fetching cluster information using Kafka admin client.
+
+|[[schema-history-internal-kafka-create-timeout-ms]]<<schema-history-internal-kafka-create-timeout-ms,`schema.history.internal.kafka.create.timeout.ms`>>
+|30
+|The number of milliseconds to wait while create kafka history topic using Kafka admin client.
+|===
```

Comment:
We need to add pass-thorough properties too, for more details for example https://debezium.io/documentation/reference/3.1/connectors/mysql.html#debezium-mysql-pass-through-database-history-properties-for-configuring-producer-and-consumer-clients